main.html

```diff
@@ -1,138 +1,171 @@
 <!DOCTYPE html>
 
 <html lang="en">
+  <head>
+    <title>ToDo List</title>
+    <link rel="stylesheet" href="./css/main.css" />
 
-    <head>
-        <title>ToDo List</title>
-        <link rel="stylesheet" href="main.css">
+    <link rel="icon" type="image/x-icon" href="./img/black-logo.PNG" />
 
-        <link rel="icon" type="image/x-icon"
-            href="./img/black-logo.PNG">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
 
-        <link rel="stylesheet"
-            href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width" />
 
-        <meta name="viewport" content="width=device-width">
+    <meta charset="utf-8" />
+  </head>
 
-        <meta charset=utf-8>
-    </head>
+  <body>
 
-    <body>
-        <div class="backgroung-box">
-            <!-- nav bar -->
-            <div class="nav-bar">
-                <div class="logo-box">
-                    <img class="logo" src="./img/black-logo.PNG" alt="todo
-                        logo" />
-                </div>
+    <main class="background-box">
 
-                <button type="button" class="nav-bar-button">
-                    <img class="button-icon" src="./img/today-icon.png"
-                        alt="today logo" /> <span>Today</span>
-                </button>
+      <nav class="nav-bar">
+        <div class="logo-box">
+         <h1>
+          <img
+            class="logo"
+            src="./img/black-logo.PNG"
+            alt="todo
+                        logo"
+          />
+         </h1>
+        </div>
 
-                <button type="button" class="nav-bar-button">
-                    <img class="button-icon" src="./img/category-icon.png"
-                        alt="category logo" /> <span>Category</span>
-                </button>
+        <button type="button" class="nav-bar-button">
+          <img
+            class="button-icon"
+            src="./img/today-icon.png"
+            alt="today logo"
+          />
+          Today
+        </button>
+
+        <button type="button" class="nav-bar-button">
+          <img
+            class="button-icon"
+            src="./img/category-icon.png"
+            alt="category logo"
+          />
+          Category
+        </button>
+
+        <button type="button" class="nav-bar-button">
+          <img class="button-icon" src="./img/goal-icon.png" alt="goals logo" />
+          Goals
+        </button>
+      </nav>
+
+      <div class="main-box">
+        <!-- task table -->
+        <div class="task-table-box">
+          <h2>My Task List</h2>
+
+          <div class="table-box">
+            <table>
+              <tr>
+                <th class="color-tag-header"></th>
+                <th>Task</th>
+                <th>Time</th>
+                <th></th>
```

Comment:
Would be nice if you could deploy this on netlify (or whatever) so we can see the design as well. I'm not sure if `<th></th>` is necessary.